rest/adminapitest/admin_api_test.go

```diff
@@ -3016,7 +3014,7 @@ func TestDbOfflineConfigLegacy(t *testing.T) {
 	// Get config values before taking db offline
 	resp = rt.SendAdminRequest("GET", "/db/_config", "")
 	require.Equal(t, http.StatusOK, resp.Code)
-	dbConfigBeforeOffline := string(resp.BodyBytes())
+	dbConfigBeforeOffline := resp.Body.String()
```

Comment:
Prefer `resp.BodyString()` and `resp.BodyBytes()` otherwise a single read like `resp.Body.String()` will clear from subsequent callers.

```suggestion
	dbConfigBeforeOffline := resp.BodyString()
```